api/db/migrations/20200503193240_add_lastTermsOfServiceValidatedAt_to_user.js

```diff
@@ -0,0 +1,15 @@
+const TABLE_NAME = 'users';
+const COLUMN_NAME = 'lastTermsOfServiceValidatedAt';
```

Comment:
Juste pour échanger, pas de changement requis
Google utilise le verbe "accept" et non pas "validate"
["To use our services, you must **accept** these terms"](https://policies.google.com/terms?hl=en-US#toc-about)

Il existe une abbréviation de Terms of Service [ToS](https://en.wiktionary.org/wiki/TOS)